test/client/fixtures/is-focusable-test.js

```diff
@@ -0,0 +1,98 @@
+var domUtils      = hammerhead.utils.dom;
```

Comment:
Please move the test file from `test/client/fixtures/` to `test/client/fixtures/utils/`